test/Utils.purs

```diff
@@ -36,22 +39,21 @@ import PureScript.CST.Types as CSTT
 
 spawnFromParent :: String -> Array String -> Aff Unit
 spawnFromParent command args = makeAff \k -> do
-  childProc <- ChildProcess.spawn command args defaultSpawnOptions { stdio = inherit }
-  ChildProcess.onExit childProc case _ of
+  childProc <- ChildProcess.spawn' command args (_ { appendStdio = Just [ inherit ] })
+  childProc # on_ ChildProcess.exitH case _ of
     Normally code
-      | code > 0 -> Process.exit code
+      | code > 0 -> Process.exit' code
       | otherwise -> k (Right unit)
-    BySignal _ ->
-      Process.exit 1
+    BySignal _ -> Process.exit' 1
   pure $ effectCanceler do
-    ChildProcess.kill SIGABRT childProc
+    void $ ChildProcess.killSignal SIGABRT childProc
 
 execWithStdin :: String -> String -> Aff ExecResult
 execWithStdin command input = makeAff \k -> do
-  childProc <- ChildProcess.exec command defaultExecOptions (k <<< pure)
-  _ <- Stream.writeString (ChildProcess.stdin childProc) UTF8 input mempty
-  Stream.end (ChildProcess.stdin childProc) mempty
-  pure $ effectCanceler $ ChildProcess.kill SIGABRT childProc
+  childProc <- ChildProcess.exec' command identity (k <<< pure)
+  _ <- Stream.writeString' (ChildProcess.stdin childProc) UTF8 input mempty
+  Stream.end' (ChildProcess.stdin childProc) mempty
+  pure $ effectCanceler $ void $ ChildProcess.killSignal SIGABRT childProc
```

Comment:
Shouldn't this be updated to use `node-execa`?

Reply:
Some of these changes are over my head and I'd just like to get the patch through for now - we can refactor more in depth in later patches

Reply:
That would be great, thanks!